cms/djangoapps/contentstore/tests/test_i18n.py

```diff
@@ -94,17 +95,19 @@ class wrap_ugettext_with_xyz(object):  # pylint: disable=invalid-name
 
             def __init__(self, module):
                 self.module = module
-                self.old_ugettext = module.ugettext
+                self.old_ugettext = module.gettext if six.PY3 else module.ugettext
```

Comment:
See the existing utility function for this in [openedx/core/lib/edx_six.py](https://github.com/edx/edx-platform/blob/master/openedx/core/lib/edx_six.py) - `get_gettext()`.